packaging/rpm/microshift.spec

```diff
@@ -19,14 +19,14 @@
 %global selinuxtype targeted
 %define selinux_policyver 3.14.3-67
 %define microshift_relabel_files() \
-   mkdir -p /var/hpvolumes; \
    mkdir -p /var/run/kubelet; \
    mkdir -p /var/lib/kubelet/pods; \
    mkdir -p /var/run/secrets/kubernetes.io/serviceaccount; \
-   restorecon -R /var/hpvolumes; \
+   mkdir -p /var/lib/microshift-backups; \
```

Comment:
What about `/var/lib/microshift`? Should we also create and selinux-tag it here?

Reply:
That's actually created by MicroShift at startup, so no worries there, it should be captured by the rules and we'll have the e2e tests in place to make sure the dirs are labeled correctly

Reply:
A comment inline about why we need to have the RPM create the backup directory to avoid the race condition would be good.

Reply:
Done, thanks!